docs/assets/style.css

```diff
@@ -212,7 +212,7 @@ span.beta {
   margin: 1em 0 0 33%;
   padding-bottom: 2em;
   font-size: 1em;
-  line-height: 1.5em;
```

Comment:
There's some funky line-height stuff happening. I don't know why this was ever 1.5 because that's not how other docs are.

Reply:
Ooof it's still doing it when it builds on github pages. I'll look into styling

Reply:
@shaydewael Sorry, I didn't mention this but I made the changes. I'd like to address this line-spacing issue in the tutorial docs. I'm not sure why it doesn't happen to Bolt for JS. Can you adjust this or keep this change as-is?

### The issue

<img width="658" src="https://user-images.githubusercontent.com/19658/94211852-8d63b100-ff0d-11ea-97e9-d13dafc3b5f6.png">


### Current

<img width="650" src="https://user-images.githubusercontent.com/19658/94211863-92286500-ff0d-11ea-8d27-ab3f03f0cd67.png">

Reply:
>Ooof it's still doing it when it builds on github pages. I'll look into styling

@shaydewael I have been observing the configuration of Jekyll in this project may have some issues. Sometimes I have to delete `_site` directory to completely refresh the pages.

Reply:
Hey Kaz - I changed a lot of styling stuff when I started messing around with this. One thing I learned - the stylesheet is being included from `slack.dev` rather than `shaydewael.github.io`. This is because it's using the `url` from the config file.

I added line numbers and adjusted spacing:
![image](https://user-images.githubusercontent.com/3411005/94486881-e729dc80-0194-11eb-9f8b-ec2990a1e2ae.png)
![image](https://user-images.githubusercontent.com/3411005/94487253-8ea70f00-0195-11eb-9c8d-8fa4fd8c4d08.png)

Reply:
This is great! Thanks a lot.